### ISSUE TITLE
High system load alert fails to trigger on 1.19.2

### ISSUE THREAD

**Issue by dnoland1** [rancher/rancher#29292]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
1. Create a k3s v1.19.2 two node cluster. Used 2 VMs with 2vCPU and 8GB RAM.
2. Import the cluster into Rancher.
3. Enable monitoring
4. Create a Notifier
5. Edit the alert group for High CPU Load alert rule and add the notifier
6. Create a workload that causes high cpu load on the cluster. I used the Folding at Home App in the Rancher catalog and set the CPU limit to 1500m 

**Result:**
The alert is never triggered, even though monitoring graphs showed CPU load well above 1.

**Other details that may be helpful:**
Dug a little more into this and saw that k3s 1.18 clusters are exporting the `machine_cpu_cores` metric but k3s 1.19 clusters are not.

k3s 1.18.8:
![image](https://user-images.githubusercontent.com/468462/94209011-19c79280-fe80-11ea-9a5f-9e2e19e45e14.png)
k3s 1.19.2:
![image](https://user-images.githubusercontent.com/468462/94209037-2946db80-fe80-11ea-8e2e-8203b832d4df.png)

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
v2.4.8
- Installation option (single install/HA):
HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported):
Imported
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
AWS EC2 m5a.large (2vCPU, 8GB RAM)

- Kubernetes version (use `kubectl version`):
```
> kubectl version
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.6", GitCommit:"d32e40e20d167e103faf894261614c5b45c44198", GitTreeState:"clean", BuildDate:"2020-05-20T13:16:24Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.2+k3s1", GitCommit:"d38505b124c92bffd45f6e0654adb9371cae9610", GitTreeState:"clean", BuildDate:"2020-09-21T17:00:07Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker version (use `docker version`):
NA, containerd 1.4.0

gzrancher/rancher#12448

**Comment by cjellick** [comment]
relinking with upstream issue bc i transfered and i think it dropped the link
https://github.com/kubernetes/kubernetes/issues/95204

**Comment by dnoland1** [comment]
On k3s 1.18.10 and 1.19.2 clusters, I was able to use `kube_node_status_capacity_cpu_cores` instead of `machine_cpu_cores`. Made a PRs to change this.

**Comment by dnoland1** [comment]
Believe the fix I made will only address new installs. For v2.4 -> v2.5 upgrades, metric will need to be manually changed.

**Comment by aiyengar2** [comment]
> On k3s 1.18.10 and 1.19.2 clusters, I was able to use `kube_node_status_capacity_cpu_cores` instead of `machine_cpu_cores`. Made a PRs to change this.

Left a comment on https://github.com/rancher/rancher/pull/29860#discussion_r529869576 about how this metric is deprecated in kube-state-metrics v2.x. I think it should be fine to use it since there's no plan of upgrading the kube-state-metrics version afaik for Alerting V1 (note: this doesn't impact Alerting V2 at all).

**Comment by jiaqiluo** [comment]
The bug is reproduced on rancher v2.5.3
Steps:
- run rancher:v2.5.3 single install 
- add a 1.19 cluster
- enable the cluster monitoring 0.1.4
- add a Slack notifier 
- use the notifier for the cluster alert group `A set of alerts for node`
- for the alert rule `High CPU load` change the threshold from 100 to 20, so we can trigger the alert easier 
- install some apps to consume a lot of CPUs

Results:
- notice that the expression is 
```
sum(node_load1) by (node)  / sum(machine_cpu_cores) by (node) * 100
```
- notice that there is not a preview of the metrics and the threshold line, which indicates that the expression is actually 

![image](https://user-images.githubusercontent.com/6218999/101962219-cdf7d000-3bc8-11eb-857c-060f315c7463.png)

- no alert is sent out when the CPU usage is above the threshold

**Comment by jiaqiluo** [comment]
The bug fix is validated on rancher:v2.5-head baf486

The steps are the same as my previous comment. 

Results:
- the new expression is 
```
sum(node_load1) by (node)  / sum(kube_node_status_capacity{resource="cpu"}) by (node) * 100
```
- there is a preview of the metric
![image](https://user-images.githubusercontent.com/6218999/101962428-61c99c00-3bc9-11eb-8614-6e4d9d316ba1.png)

- and the alert message is sent out to Slack when it is triggered
